tests/unit/test_hardware.py

```diff
@@ -257,9 +259,11 @@ def test_parse_maximal_constraint() -> None:
           - and:
               - and:
                   - disk[0].size: == 40 GiB
+                  - disk[0].physical-sector-size: == 4096 dimensionless
```

Comment:
Now this is unexpected - it's not dimensionless, right? What should be the unit, bytes?

1. the examples should include units,
2. the specification does not mention units anywhere, `disk.size` and `memory` do not mention the default unit, because there is none, which is not correct, so let's keep sector size also not mentioning unit in its specification, just examples, and
3. in a dedicated patch, we should add a bit of code to `tmt.hardware`, to coerce dimensionless values of `SizeConstraint` to values with dimensions. Something like a default unit passed to `SizeConstraint.from_specification`, is to be used for dimensionless values. So far it seems the default would be `bytes`, could be a keyword argument with the `bytes` default. Together with this addition, we would add a note to HW requirements specification, to `memory` and `disk.size` and sector sizes, "Hey, the default unit is bytes".

What do you think?